### PR TITLE
WIP: CSS Styling and NavBar

### PR DESCRIPTION
**Pull request by whitneyfirth**
Added styling details such as the search input field, carousels, color theme, and the popup cards. Also added the Home/Board games/Card games titles in the nav bar which still need to have functionality added to them (WIP). 

All colors/styles are still totally up for changes, I just had fun playing around with it all!

### CODE REVIEW

File: src/styles/_Navbar.scss [hunk 1]
```diff
@@ -1,23 +1,69 @@
 .Navbar {
   display: flex;
   justify-content: space-between;
-  height: 10%;
-  background-color: #2F363D;
+  height: 75px;
```
Comment (JustinD85) [inline src/styles/_Navbar.scss:4]: I am curious why fixed height here? The nav and main area I thought would take up a set % height of the screen. 10% and 90% respectively. Were there issues with that setup?

File: src/styles/_Carousel.scss [hunk 2]
```diff
@@ -25,9 +34,10 @@
 }
 
 .game-card {
```
Comment (JustinD85) [inline src/styles/_Carousel.scss:36]: When I search for a title, the fewer titles there are the taller the cards become. Not sure what direction to take to fix. Maybe a max-height combined with % height? height 280px max-height:10%?

File: src/styles/_Navbar.scss (hunk 1 above)
Comment (whitneyfirth) [reply src/styles/_Navbar.scss:4]: Resolved through Slack.

File: src/styles/_Carousel.scss (hunk 2 above)
Comment (whitneyfirth) [reply src/styles/_Carousel.scss:36]: Yes you're right! I didn't catch that. Tomorrow lets try either all fixed heights or all percentages.

**Pull request merged**